tests/unit/test_charm.py

```diff
@@ -68,7 +68,7 @@ def test_reconcile_reaches_active_status_when_repository_configured(
     wazuh_install_certificates_mock.assert_called_with(container, ANY, "somecert")
     wazuh_update_configuration_mock.assert_called_with(container, ["10.0.0.1"])
     configure_git_mock.assert_called_with(
-        container, wazuh_config.custom_config_repository, "somekey"
+        container, str(wazuh_config.custom_config_repository), "somekey"
```

Comment:
Do we actually need to cast it to a string or do we just need a typing.cast for mypy?

Reply:
We need to cast it, it crashes otherwise